molgenis-data-import/src/main/java/org/molgenis/data/importer/emx/EmxMetadataParser.java

```diff
@@ -968,44 +968,36 @@ void validateAutoAttrValue(int rowIndex, Attribute attr, String emxEntityName) {
   void setRange(
       Entity emxAttrEntity, String emxEntityName, String emxName, Attribute attr, int rowIndex) {
     String emxRangeMin = emxAttrEntity.getString(EMX_ATTRIBUTES_RANGE_MIN);
+    Long rangeMin =
+        toRange(emxEntityName, emxName, rowIndex, emxRangeMin, EMX_ATTRIBUTES_RANGE_MIN);
+
+    String emxRangeMax = emxAttrEntity.getString(EMX_ATTRIBUTES_RANGE_MAX);
+    Long rangeMax =
+        toRange(emxEntityName, emxName, rowIndex, emxRangeMax, EMX_ATTRIBUTES_RANGE_MAX);
+
+    if (rangeMin != null || rangeMax != null) {
+      attr.setRange(new Range(rangeMin, rangeMax));
+    }
+  }
+
+  private Long toRange(
+      String emxEntityName,
+      String emxName,
+      int rowIndex,
+      String emxRangeMin,
+      String emxAttributesRangeMin) {
     Long rangeMin;
```

Comment:
Variables in this method should just be called `range` I think because it's also being used to convert to `rangeMax`.